source/FpsConfig.cpp

```diff
@@ -200,6 +200,65 @@ void PlayerConfig::load(FPSciAnyTableReader reader, int settingsVersion) {
 		reader.getIfPresent("movementRestrictionZ", movementRestrictionZ);
 		reader.getIfPresent("restrictedMovementEnabled", restrictedMovementEnabled);
 		reader.getIfPresent("counterStrafing", counterStrafing);
+		reader.getIfPresent("propagatePlayerConfigs", propagatePlayerConfigsToAll);
+
+		clientPlayerConfigs.push_back(PlayerConfig());
+		clientPlayerConfigs.push_back(PlayerConfig());
+
+		// For Client 1
+		reader.getIfPresent("Client1moveRate", clientPlayerConfigs[0].moveRate);
```

Comment:
I would strongly suggest avoiding these hard-coded client-specific config fields in favor of directly serializing in `clientPlayerConfigs` as an `Array<PlayerConfig>` from Any. You may need to add an `Any` constructor for `PlayerConfig` for this to work (essentially just a pass through to calling `PlayerConfig::load()`)

Reply:
I have no experience in working with .any files as of now. Also, this seems like it will take time (for me) to get right. Will keep this for the next iteration.